### ISSUE TITLE
Fast evaluation across multiple sets of regions.

### ISSUE THREAD

**Issue by jmonlong** [jmonlong/sveval#8]
Right now there is a parameter to define regions to run the evaluation on. Basically the overlap are computed and then the SVs are filtered to keep only the ones within these regions. We used this to test the performance in non-repeat regions (or other high-confidence regions). 

So currently we run the whole thing twice, whole-genome and in non-repeat regions, which is not that bad. This would be quite inefficient if we tried to evaluate on many sets of regions, like for example in different classes of repeats. 

A new function, very similar to the per-size function *plot_persize*, could do this more efficiently from the output (FP, FN, TP with best F1 score). The quality threshold wouldn't be optimized for each region set (it would be using the one maximizing the F1 score overall) but that not problem when exploring how the genotyper did on different regions.

In summary, using the input parameter *bed.regions* can be used to find out what's the best that the genotyper can do on a specific set of regions; the new function will just zoom in different regions for a particular FP/TP/FN set (using a fixed QUAL threshold).
